docs/Supported-Platforms.md

```diff
@@ -23,7 +23,7 @@ recommendations on which to run Brim:
    * Windows 10 or newer
    * Windows Server 2019 or newer
 * macOS
-   * macOS Catalina 10.15.7 or newer (see [below](#hardware) for hardware considerations)
+   * macOS Big Sur 11.6.8 or newer (see [below](#hardware) for hardware considerations)
```

Comment:
FYI, while GItHub just claims "macOS 11" in their docs, I grabbed the full version string by executing `sw_vers` on a `macos-11` runner. I'm sure this will be subject to change as Apple keeps releasing minor updates.